actors/builtin/init/init_test.go

```diff
@@ -1 +1,154 @@
 package init_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ipfs/go-cid"
+	assert "github.com/stretchr/testify/assert"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	builtin "github.com/filecoin-project/specs-actors/actors/builtin"
+	_init "github.com/filecoin-project/specs-actors/actors/builtin/init"
```

Comment:
Nit: I'd prefer `init_` so that things like IDE autocomplete are more likely to be useful.